src/Extensions/Constants/ShopUrls.php

```diff
@@ -79,6 +90,11 @@ private function init($lang)
 
     public function returns($orderId, $orderAccessKey = null)
     {
+        if($orderAccessKey == null) {
+            $request = pluginApp(Request::class);
+            $orderAccessKey = $request->get('accessKey');
+        }
+
         return $this->getShopUrl(RouteConfig::ORDER_RETURN, "returns", $orderId, $orderAccessKey);
```

Comment:
Bei der Überführung der OrderConfirmation in den ShopBuilder mussten wir die OrderID und den AccessKey wieder aus der URL in Query-Parameter auslagern. Bitte nochmal mit @Dominik2809 klären, ob das hier auch notwendig sein sollte.

Reply:
Meyer prüft das

Reply:
Wie besprochen sind hier noch nicht alle Fälle mit abgedeckt, da der accessKey nicht zwangsläufig in der URL steht. Klickt ein Gast auf "Kaufen", wird er auf die Confirmation geleitet, ohne dass sich der accessKey in der URL befindet.